src/test/e2e/psp.feature.ts

```diff
@@ -0,0 +1,353 @@
+    deployer: new hre.ethers.Wallet(localConfig.privateKeys.deployer).connect(provider),
+    manager: new hre.ethers.Wallet(localConfig.privateKeys.manager).connect(provider),
+    sponsor: new hre.ethers.Wallet(localConfig.privateKeys.sponsor).connect(provider),
+    randomPerson: new hre.ethers.Wallet(localConfig.privateKeys.randomPerson).connect(provider),
+  };
+
+  const dapiServerAdminRoleDescription = 'DapiServer admin';
+  let accessControlRegistryAbi;
+  let accessControlRegistryFactory: ContractFactory;
+  let accessControlRegistry: Contract;
+  let airnodeProtocolAbi;
+  let airnodeProtocolFactory: ContractFactory;
+  let airnodeProtocol: Contract;
+  let dapiServerAbi;
+  let dapiServerFactory: ContractFactory;
+  let dapiServer: Contract;
+
+  beforeEach(async () => {
+    //Reset the local hardhat network state for each test to keep the deployed Airnode and DapiServer contract addresses
+    //the same as the config files
+    await hre.network.provider.send('hardhat_reset');
+
+    jest.restoreAllMocks();
+
+    // Deploy contracts
+    accessControlRegistryAbi = JSON.parse(
+      fs.readFileSync(path.resolve('./scripts/artifacts/AccessControlRegistry.json')).toString()
+    );
+    accessControlRegistryFactory = new hre.ethers.ContractFactory(
+      accessControlRegistryAbi.abi,
+      accessControlRegistryAbi.bytecode,
+      roles.deployer
+    );
+    accessControlRegistry = await accessControlRegistryFactory.deploy();
+    airnodeProtocolAbi = JSON.parse(
+      fs.readFileSync(path.resolve('./scripts/artifacts/AirnodeProtocol.json')).toString()
+    );
+    airnodeProtocolFactory = new hre.ethers.ContractFactory(
+      airnodeProtocolAbi.abi,
+      airnodeProtocolAbi.bytecode,
+      roles.deployer
+    );
+    airnodeProtocol = await airnodeProtocolFactory.deploy();
+
+    dapiServerAbi = JSON.parse(fs.readFileSync(path.resolve('./scripts/artifacts/DapiServer.json')).toString());
+    dapiServerFactory = new hre.ethers.ContractFactory(dapiServerAbi.abi, dapiServerAbi.bytecode, roles.deployer);
+    dapiServer = await dapiServerFactory.deploy(
+      accessControlRegistry.address,
+      dapiServerAdminRoleDescription,
+      roles.manager.address,
+      airnodeProtocol.address
+    );
+
+    // Access control
+    const managerRootRole = await accessControlRegistry.deriveRootRole(roles.manager.address);
+    await accessControlRegistry
+      .connect(roles.manager)
+      .initializeRoleAndGrantToSender(managerRootRole, dapiServerAdminRoleDescription);
+
+    // Wallets
+    const airnodeWallet = hre.ethers.Wallet.fromMnemonic(localConfig.airnodeMnemonic);
+    const airnodePspSponsorWallet = node.evm
+      .deriveSponsorWalletFromMnemonic(localConfig.airnodeMnemonic, roles.sponsor.address, PROTOCOL_ID_PSP)
+      .connect(provider);
+    await roles.deployer.sendTransaction({
+      to: airnodePspSponsorWallet.address,
+      value: hre.ethers.utils.parseEther('1'),
+    });
+
+    // Setup ETH Subscription
```

Comment:
Mhh, does this comment refer to something?

Reply:
Yes so I added this for it to be clear that the following lines are for setting up the ETH subscription (i.e. `buildLocalConfig`)

Maybe these should be renamed to `buildLocalConfigETH` and `buildLocalConfigBTC` to be clear